packages/block-library/src/image/edit.js

```diff
@@ -145,7 +145,7 @@ export function ImageEdit( {
 				scale: undefined,
 			} );
 		}
-	}, [ align ] );
+	}, [ __unstableMarkNextChangeAsNotPersistent, align, setAttributes ] );
```

Comment:
I decided to fix both warnings in the file. Both callbacks have stable references and won't change the effect's behavior.

Reply:
Thanks for the fixes 🙌